fighthealthinsurance/nice_tools.py

```
@@ -0,0 +1,349 @@
+            )
+
+        if guidance_models:
+            try:
+                await NICEQueryData.objects.acreate(
+                    denial_id=denial,
+                    query=query,
+                    results=json.dumps(
+                        [g.guidance_id for g in guidance_models]
+                    ).replace("\x00", ""),
+                )
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)